README.md

```diff
@@ -31,3 +31,21 @@ Files and directories important to the structure of the upstream include:
 * `composer.json`: The composer file at the project root is for project settings and scaffolding. Vendor-supported sites or sites with emergent features may add dependencies or customizations here. Developers should avoid updating this file in the upstream.
 * `pantheon.upstream.yml`: The build_step: true directive in pantheon.upstream.yml enables the build step.
 * `README.md`: You are [here](#).
+
+## Local Development Environment
+
+This project supports development with Lando using the Pantheon recipe. Most local tooling, including Composer, Drush, Drupal console, and node applications are available through this recipe.
+
+### Requirements
+
+* [Composer](https://getcomposer.org/download/): PHP package manager. Version 2.x.
+* [Terminus](https://pantheon.io/docs/terminus): Pantheon CLI.
+* [Docker](https://docs.docker.com/install): Container system for virtualization.
+* [Lando](https://docs.lando.dev/basics/installation.html#system-requirements): Recipes and tools for Docker.
+
+### Setup
+* [Lando CA](https://docs.devwithlando.io/config/security.html): Optional certificate authority setup.
+* Public key for your machine uploaded to Pantheon.
```

Comment:
If we have adding the public key as a setup step, we might as well suggest developers to create a machine token for using Terminus.